src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
+
+	@Override
+	public boolean checkValidity(CreditCardVendor ccv, String number) {
```

Comment:
CreditCardValidator  will become eventually a rest service ,checkValidity should take as argument :vendor name or ID of vendor, this argument should be validated against supported vendors, card number should be validated against card number required length. Both arguments should be validated against null. Validity check should be  moved to specific vendor implementation, as separated class